test/test.js

```diff
@@ -331,11 +331,11 @@ describe('parseForecast_simple_caaml caic', function(){
         })
     })
 })
-
+/* DISABLE BTAC test since it is now on NAC parser
```

Comment:
@andyanderso you can just delete this block of code, thanks

Reply:
actually i guess we've just been commenting them out. fine for now. it looks like the other PR just opened deletes a bunch of this dead code.